lnbits/extensions/admin/migrations.py

```diff
@@ -0,0 +1,74 @@
+async def m001_create_admin_settings_table(db):
+    await db.execute(
+        """
+        CREATE TABLE IF NOT EXISTS admin.settings (
+            lnbits_admin_ui TEXT,
```

Comment:
- `lnbits_admin_ui` - this value will never be used since `settings` will be initialized with `LNBITS_ADMIN_UI`

Reply:
removed it, and also some other values that should not saved into db.